pom.xml

```diff
@@ -326,5 +326,22 @@
             <artifactId>json-schema-validator</artifactId>
             <version>1.0.55</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.github.rholder</groupId>
+            <artifactId>guava-retrying</artifactId>
+            <version>2.0.0</version>
+        </dependency>
+
+        <dependency>
```

Comment:
Please scope the uses of org.mockito dependencies to test only.

Reply:
Also, please bump the version to 1.0.4

Reply:
Good catch on the test scope. Fixed in bca1ff6.

Just for cleanliness, I lean toward reserving version bumps to separate PRs. WDYT?

Reply:
That's fine. I'm going to call this one good then.